azext_iot/_params.py

```diff
@@ -378,3 +385,70 @@ def load_arguments(self, _):
+
+    with self.argument_context('iot dt monitor-events') as context:
+        context.argument('repair', options_list=['--repair'],
+                         arg_type=get_three_state_flag(),
+                         help='Reinstall uamqp dependency compatible with extension version.')
+        context.argument('consumer_group', options_list=['--consumer-group', '--cg'],
+                         help='Specify the consumer group to use when connecting to event hub endpoint.')
+        context.argument('properties', options_list=['--properties', '--props', '-p'], arg_type=event_msg_prop_type)
+
+    with self.argument_context('iot pnp') as context:
+        context.argument('model', options_list=['--model', '-m'],
+                         help='Target capability-model urn-id. Example: urn:example:capabilityModels:Mxchip:1')
+        context.argument('interface', options_list=['--interface', '-i'],
+                         help='Target interface urn-id. Example: urn:example:interfaces:MXChip:1')
+
+    with self.argument_context('iot pnp interface') as context:
+        context.argument('interface_definition', options_list=['--definition', '--def'],
+                         help='IoT Plug and Play interface definition written in PPDL (JSON-LD). '
+                         'Can be directly input or a file path where the content is extracted.')
+
+    with self.argument_context('iot pnp interface list') as context:
+        context.argument('search_string', options_list=['--search', '--ss'],
+                         help='Searches IoT Plug and Play interfaces for given string in the'
+                              ' \"Description, DisplayName, comment and Id\".')
+        context.argument('page_size', options_list=['--page', '--ps'],
+                         help='Page Size.')
+        context.argument('token', options_list=['--token', '-t'],
+                         help='Continuation Token.')
+
+    with self.argument_context('iot pnp capability-model') as context:
+        context.argument('model_definition', options_list=['--definition', '--def'],
+                         help='IoT Plug and Play capability-model definition written in PPDL (JSON-LD). '
+                         'Can be directly input or a file path where the content is extracted.')
+
+    with self.argument_context('iot pnp capability-model show') as context:
+        context.argument('expand', options_list=['--expand'],
+                         help='Indicates whether to expand the device capability model\'s'
+                              ' interface definitions or not.')
+
+    with self.argument_context('iot pnp capability-model list') as context:
+        context.argument('search_string', options_list=['--search', '--ss'],
+                         help='Searches IoT Plug and Play models for given string in the'
+                              ' \"Description, DisplayName, comment and Id\".')
+        context.argument('page_size', options_list=['--page', '--ps'],
```

Comment:
Same comment as above regarding `page_size`. Take a look at how the iot hub query functionality [is implemented](https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/operations/generic.py). Token and page size is abstracted from the end user. A 'top' argument is leveraged instead.

Reply:
replace "top" in place of page_size and token